test/unit/server_test.js

```diff
@@ -255,4 +280,148 @@ describe("server.js tests", function () {
       expect(typeof json.test).to.be.equal("function");
     });
   });
+
+  describe("Smoke tests for the rest of the builders", function() {
+    describe("AccountCallBuilder", function() {
+      beforeEach(function (done) {
+        // instruct the dev server to except the correct request
+        return this.setFixtures({
+          request: "/accounts/GSDEF?limit=4",
+          response: {status: 200, body: FAKE_COLLECTION_RESPONSE}
+        }).then(function () { done() });
+      });
+
+      it("requests the correct endpoint", function (done) {
+        server.accounts()
+          .address("GSDEF")
+          .limit("4")
+          .call()
+          .then(function () {
+            done();
```

Comment:
We're not checking a response the `call` method's promise returns so we're not testing whether the URL built is correct (it this example it should be `/accounts/GSDEF?limit=4`).

One way to do this is to tell the fake server what to respond for specific request. The other way is to mock `_sendNormalRequest` method ([sinon](http://sinonjs.org/docs/#mocks) is great for mocking) and check if it's called with correct URL.

Or you can leave tests in `server_test.js` as they are and write unit tests for builders.